src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -132,7 +132,8 @@ interface IOptionSettlementEngine {
         uint96 underlyingAmount,
         uint40 exerciseTimestamp,
         uint40 expiryTimestamp,
-        uint96 nextClaimId
+        uint96 nextClaimId,
+        uint40 creationTimestamp
```

Comment:
We should see if we can pack these better.

Reply:
Created #70 to track this